Candidate: CVE-2022-30284
PublicDate: 2022-05-04 22:15:00 UTC
References:
 https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-30284
 https://www.swascan.com/security-advisory-libnmap-2/
 https://github.com/savon-noir/python-libnmap/releases
 https://pypi.org/project/python-libnmap/
Description:
 In the python-libnmap package through 0.7.2 for Python, remote command
 execution can occur (if used in a client application that does not validate
 arguments).
Ubuntu-Description:
Notes:
Mitigation:
Bugs:
Priority: medium
Discovered-by:
Assigned-to:
CVSS:


Patches_python-libnmap:
upstream_python-libnmap: needs-triage
focal_python-libnmap: needs-triage
impish_python-libnmap: needs-triage
jammy_python-libnmap: needs-triage
devel_python-libnmap: needs-triage
